react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+const listItems = pokedex.map(pokemon =>
```

Comment:
Although this will work it is best practice to use parentheses around the returned value of the arrow function:

```JSX
const listItems = pokedex.map(pokemon => (
  <li key={pokemon.number}>
    {pokemon.name}
  </li>
));
```